Libraries/ReactNative/HeadlessJsTask.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+'use strict';
+
+export class HeadlessJsTaskError extends Error {}
```

Comment:
Can you change the file name to HeadlessJsTaskError to match the class? Also, instead of putting it at the top level, can you put it /Utilities, maybe?